doc/Tex/RAM_SCB_quickstart.tex

```diff
@@ -1,38 +1,23 @@
-This section provides a bare-bones approach for getting started with RAM-SCB. 
-The instructions are aimed at users familiar with Unix-like environments.
-Tutorial-like guides for each step can be found in other chapters.
+This section provides a bare-bones approach for getting started with RAM-SCB. The instructions are aimed at users familiar with Unix-like environments. Tutorial-like guides for each step can be found in other chapters.
 
 \section{Installation}
 
-Installation and compilation of RAM-SCB requires a Fortran compiler, 
-Perl interpreter, an MPI library compiled with the preferred Fortran compiler, 
-and several external libraries which have 
-additional 
-requirements (most notably a C compiler, typically standard on Unix-like 
-systems.)  The required libraries are:
+Installation and compilation of RAM-SCB requires a Fortran compiler, Perl interpreter, an MPI library compiled with the preferred Fortran compiler, and several external libraries which have additional requirements (most notably a C compiler, typically standard on Unix-like systems.)  The required libraries are:
```

Comment:
Compiler modern enough to have OpenMP preferred...

Reply:
Yes